From 6c060a62be09e194e871d0d3b9a7b93b100ce558 Mon Sep 17 00:00:00 2001
From: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
Date: Mon, 4 Oct 2021 17:27:25 +0300
Subject: [PATCH 256/322] [FIO extras-squash] cmd: fiovb: fix using undeclared
 function boot_mode_is_closed()
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Include a header file with the declaration of boot_mode_is_closed()
function, fixing a compiler warning [1].

[1]
cmd/fiovb.c: In function ‘do_fiovb_init’:
  CC      drivers/core/ofnode.o
cmd/fiovb.c:23:7: warning: implicit declaration of function ‘boot_mode_is_closed’ [-Wimplicit-function-declaration]
   23 |  if (!boot_mode_is_closed())
      |       ^~~~~~~~~~~~~~~~~~~

Fixes: 545d964536 ("[FIO extras] fiovb: return error on init if board not closed")
Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
---
 cmd/fiovb.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmd/fiovb.c b/cmd/fiovb.c
index d11240339f8..22b970c374c 100644
--- a/cmd/fiovb.c
+++ b/cmd/fiovb.c
@@ -10,6 +10,7 @@
 #include <malloc.h>
 #include <mmc.h>
 #include <fiovb.h>
+#include <asm/arch/sys_proto.h>
 
 static struct fiovb_ops *fiovb_ops;
 
-- 
2.34.1

